src/Akeneo/CouplingDetector/NodeParser/PhpClassNodeParser.php

```diff
@@ -28,8 +28,8 @@ public function parse(\SplFileInfo $file)
 
         $content = file_get_contents($file->getRealPath());
         $tokens = Tokens::fromCode($content);
-        $classNamespace = $namespaceExtractor->extract($tokens);
-        $className = $classNameExtractor->extract($tokens);
+        $classNamespace = $namespaceExtractor->extract($tokens, $file);
```

Comment:
i would catch the ExtractionException here and throw a new ParserException indicating the failing file path.